python/lognplot/qt/widgets/chartwidget.py

```diff
@@ -109,16 +132,19 @@ def horizontal_zoom(self, amount, around):
         self.chart.horizontal_zoom(amount, around)
         # Autoscale Y for a nice effect?
         self.chart.autoscale_y()
+        self.repaint()
```

Comment:
What does this function do? update also triggers a paint action eventually. Does this result in snappier look / feel?

Reply:
The handles on the left hand side otherwise lagged behind in the repainting. Not sure about the reason though. Maybe that 'update' only invalidated the chart area itself?